external-api/api-tests/v1/party-db.api-test.ts

```diff
@@ -12,19 +12,81 @@
 
 import MockAdapter from 'axios-mock-adapter';
 import axios, { HttpStatusCode } from 'axios';
-import { MOCK_COMPANY_REGISTRATION_NUMBERS } from '@ukef/dtfs2-common';
+import { MOCK_COMPANY_REGISTRATION_NUMBERS, isAutomaticSalesforceCustomerCreationFeatureFlagEnabled } from '@ukef/dtfs2-common';
```

Comment:
```suggestion
import { MOCK_COMPANY_REGISTRATION_NUMBERS, isSalesforceCustomerCreationEnabled } from '@ukef/dtfs2-common';
```

Shorter name please.